project-lifecycle.md

```diff
@@ -118,3 +108,15 @@ period, the project will be labeled *End of Life*.
 # End of Life
 
 A project that is no longer actively developed or maintained.
+
+# Promoted Release
+
+A project\'s maintainers seeking to publish a *Promoted
+Release* (see Hyperledger\'s use of [semver](./release-taxonomy.md#1);
```

Comment:
What does semver have to do with promoted releases?

Reply:
I'm guessing this is leftover from the original text that was about First Major Release!